core/generator.rb

```diff
@@ -442,10 +443,10 @@ def Generator.nodeDefinition(node, boxes, path, cookbook_path)
           machine = getDockerDef(cookbook_path, name, template_path, provisioned)
           copyDockerfiles(path, name, platform, platform_version)
         else
-          $out.warning 'WARNING: Configuration has not support AWS, config file or other vm provision'
+          $out.warning 'Configuration has not support AWS, config file or other vm provision!'
```

Comment:
Ok, I will fix it. It means that configuration does not contain AWS config or other vm provision other then vm, aws, libvirt, docker